### PR TITLE
docs: sd-video docs [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/components/video/video.stories.ts [hunk 1]
```diff
@@ -2,228 +2,78 @@
...
-  value:
-    '<img slot="poster" alt="poster" class="w-[854px] aspect-video cover" src="./placeholders/images/architecture.jpg" />'
-};
-const iconConstant: ConstantDefinition = {
-  type: 'slot',
-  name: 'play-icon',
-  value: '<sd-icon slot="play-icon" library="system" name="start"></sd-icon>'
-};
+/**
+ * Used for embedding and controlling videos.
```
Comment (yoezlem) [inline packages/components/src/components/video/video.stories.ts:10]:
```suggestion
 * Used to embed and controll videos. 
```

File: packages/components/src/components/video/video.stories.ts [hunk 2]
```diff
@@ -2,228 +2,78 @@
...
+    <sd-video overlay>
+      <img alt="Test" class="w-[854px] aspect-video cover" src="./placeholders/images/architecture.jpg" />
+    </sd-video>
+  `
 };
 
 /**
- * Duplicates the previous story with default focus on the play button.
+ * Use in combination with a viewer component (e. g. from Moving Image) or a bare `<video>`-Tag. <br/>
+ * Use the `poster` slot to add a preview image.
```
Comment (yoezlem) [inline packages/components/src/components/video/video.stories.ts:61]:
```suggestion
 * Use in combination with a viewer component (e. g. from Moving Image) or a bare `<video>`-Tag.
 * 
 * Use the `poster` slot to add a preview image.
```

File: packages/components/src/components/video/video.stories.ts [hunk 3]
```diff
@@ -2,228 +2,79 @@
...
-          constants: [videoConstant, posterConstant]
-        })}
-        ${videoElementScript}
-      </div>
-    `;
-  }
+export const Playing = {
+  render: () => html`
+    <sd-video playing>
+      <img alt="Test" class="w-[854px] aspect-video cover" src="./placeholders/images/architecture.jpg" />
```
Comment (smfonseca) [inline packages/components/src/components/video/video.stories.ts:65]: Please update alt text

File: packages/components/src/components/video/video.stories.ts [hunk 4]
```diff
@@ -2,228 +2,79 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
-import { waitUntil } from '@open-wc/testing-helpers';
-import { withActions } from '@storybook/addon-actions/decorator';
-import type { ConstantDefinition } from '../../../scripts/storybook/helper';
 
 const { argTypes, args, parameters } = storybookDefaults('sd-video');
 const { generateTemplate } = storybookTemplate('sd-video');
```
Comment (smfonseca) [inline packages/components/src/components/video/video.stories.ts:8]: Link to the related template is missing

File: packages/components/src/templates/video-with-description-and-copyright.stories.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      </figcaption>
+    </sd-video>
+  `
+};
+
+export const VideoElementWithCopyright = {
+  render: () => html`
+    <style>
+      .sd-copyright::after {
+        z-index: 9999;
```
Comment (smfonseca) [inline packages/components/src/templates/video-with-description-and-copyright.stories.ts:37]: I think we can lower this value to 10 which is the same as the poster. 9999 was only for testing purposes.

File: packages/components/src/templates/video-with-description-and-copyright.stories.ts [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  title: 'Templates/Video with Description and Copyright',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+export const VideoElementWithDescription = {
+  render: () => html`
+    <sd-video class="sd-media">
+      <video controls="" id="video-example" class="w-[854px] aspect-video">
```
Comment (smfonseca) [inline packages/components/src/templates/video-with-description-and-copyright.stories.ts:15]: Please check if it's possible to remove the id. As far as I see there no need to have it here.

File: packages/components/src/components/video/video.stories.ts (hunk 3 above)
Comment (auroraVasconcelos) [reply packages/components/src/components/video/video.stories.ts]: Updated

File: packages/components/src/components/video/video.stories.ts (hunk 4 above)
Comment (auroraVasconcelos) [reply packages/components/src/components/video/video.stories.ts:8]: Updated

File: packages/components/src/templates/video-with-description-and-copyright.stories.ts (hunk 6 above)
Comment (auroraVasconcelos) [reply packages/components/src/templates/video-with-description-and-copyright.stories.ts]: Updated

File: packages/components/src/templates/video-with-description-and-copyright.stories.ts (hunk 5 above)
Comment (auroraVasconcelos) [reply packages/components/src/templates/video-with-description-and-copyright.stories.ts]: Updated

**Review by smfonseca — APPROVED** [review]

File: packages/components/src/components/video/video.stories.ts [hunk 7]
```diff
@@ -2,228 +2,82 @@
...
+        class="w-[854px] aspect-video cover"
+        src="./placeholders/images/architecture.jpg"
+      />
+    </sd-video>
+  `
 };
 
 /**
- * Utility script used in next 2 stories. You can use similar logic to wire up video elements in your project.
+ * Use the attribute `playing` to hide the play icon and the overlay.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/video/video.stories.ts:62]:
```suggestion
 * Use the `playing` attribute to hide the play icon and the overlay.
```

File: packages/components/src/components/video/video.stories.ts [hunk 8]
```diff
@@ -2,228 +2,82 @@
...
+  render: () => html`
+    <sd-video playing>
+      <img alt="Generic Alt" class="w-[854px] aspect-video cover" src="./placeholders/images/architecture.jpg" />
+    </sd-video>
+  `
 };
 
 /**
- * Duplicates the previous story with default focus on the play button.
+ * Use the attribute `overlay` to show a dark overlay. Only used when `playing` is `false`.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/video/video.stories.ts:74]:
```suggestion
 * Use the `overlay` attribute to show a dark overlay. Only used when `playing` is `false`.
```

**Review by MartaPintoTeixeira — APPROVED** [review]

**Pull request merged**